### PR TITLE
Update deDE.lua

### PR DESCRIPTION
**Pull request by bullsei**
translated the 5 new phrases to german

### CODE REVIEW

File: Locales/deDE.lua [hunk 1]
```diff
@@ -1,12 +1,12 @@
 local L = LibStub("AceLocale-3.0"):NewLocale("Big Wigs", "deDE")
 if not L then return end
 
---L.getNewRelease = "Your Big Wigs is old (/bwv) but you can easily update it using the Curse Client. Alternatively, you can update manually from curse.com or wowinterface.com."
---L.warnTwoReleases = "Your Big Wigs is 2 releases out of date! Your version may have bugs, missing features, or completely incorrect timers. It is strongly recommended you update."
---L.warnSeveralReleases = "|cffff0000Your Big Wigs is several releases out of date!! We HIGHLY recommend you update to prevent syncing issues with other players!|r"
+L.getNewRelease = "Dein Big Wigs ist veraltet (/bwv), aber du kannst es mit Hilfe des Curse Client einfach aktualisieren. Alternativ kannst du es auch manuell von curse.com oder wowinterface.com herunterladen und manuell aktualisieren."
```
Comment (Stanzilla) [inline Locales/deDE.lua]: You got `manuell` twice here.

File: Locales/deDE.lua [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+L.warnTwoReleases = "Dein Big Wigs ist 2 Versionen älter als die neueste Version! Deine Version könnte Fehler, fehlende Funktionen oder völlig falsche Zeitmesser beinhalten. Es wird dringend empfohlen, Big Wigs zu aktualisieren."
```
Comment (Stanzilla) [inline Locales/deDE.lua]: Please don't translate `timer` with `Zeitmesser` that's just super weird. Actually I would just not translate it at all.

File: Locales/deDE.lua [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
---L.gitHubTitle = "Big Wigs is on GitHub"
---L.gitHubDesc = "Big Wigs is open source software hosted on GitHub. We are always looking for new people to help us out and everyone is welcome to inspect our code, make contributions and submit bug reports. Big Wigs is as great as it is today largely in part to the great WoW community helping us out.\n\nIf you've committed changes to Big Wigs whilst it was on SVN and would like your GitHub account linked on the Git commit log, contact us."
+L.gitHubTitle = "Big Wigs ist auf GitHub"
+L.gitHubDesc = "Big Wigs ist Open-Source-Software auf GitHub. Wir sind immer auf der Suche nach neuen Menschen, die uns helfen, und jeder ist willkommen, unseren Code zu inspizieren, Beiträge zu leisten und Fehlerberichte einzureichen. Big Wigs ist hauptsächlich durch die großartige WoW-Gemeinschaft im Laufe der Zeit zu etwas Großem geworden.\n\nFalls du Änderungen an Big Wigs vorgenommen hast, während Big Wigs noch auf SVN war, und Du möchtest, dass dein GitHub-Konto mit Gits Commit-Protokoll verknüpft wird, kontaktiere uns."
```
Comment (Stanzilla) [inline Locales/deDE.lua]: `während Big Wigs noch auf SVN war, und Du möchtest, dass dein GitHub-Konto mit Gits Commit-Protokoll verknüpft wird, kontaktiere uns` make that something like `während Big Wigs noch SVN benutzt hat, und du deine Commits deinem GitHub Konto zugeordnet haben willst, kontaktiere uns`

**Comment by bullsei** [comment]
ok, how do i change this now? i edit the file in my patch1 branch and send a new pull request? is it actually a good idea for me to translate since it seems like you speak german as well?

**Comment by nebularg** [comment]
All our germans aren't german enough to actually be useful translators, they're just bullies :(

If you commit your changes and push to the branch again, they'll show up here. Thanks!

**Comment by bullsei** [comment]
Ok, i thought Stanzilla is german because his corrections were actually pretty good german :)
Ok i will try that, damn you guys are fast responders.

**Comment by Stanzilla** [comment]
Thank you!

**Pull request merged**